doc/docs/deploy-manage/deploy/environment-variables.md

```diff
@@ -0,0 +1,166 @@
+# Configure Environment Variables
+
+When you use Pachyderm, you can define environment variables that
+can transmit the required configuration directly to your application.
+
+In Pachyderm, you can define the following types of environment
+variables:
+
+* `pachd` environment variables that define parameters for your
+Pachyderm daemon container.
+
+* Pachyderm worker environment variables that define parameters
+on the Kubernetes pods that run your pipeline code.
+
+You can configure environment variables as you typically do
+in a Linux environment. For example, to configure a `PACH_JOB_ID`
+in bash, run:
+
+```bash
+$ export PACH_JOB_ID=<ID>
+```
+
+You can access all the variables in the Pachyderm manifest that
+is generated when you run `pachctl deploy` with the --dry-run`
+flag.
+
+!!! note "See also"
+    [Deploy Pachyderm](../../../getting_started/local_installation/#deploy-pachyderm)
+
+## `pachd` Environment Variables
+
+You can find the list of `pachd` environment variables in the
+`pachd` manifest by running the following command:
+
+```bash
+$ kubectl get deploy pachd -o yaml
+```
+
+The following tables list all the `pachd`
+environment variables.
+
+**Global Configuration**
+
+| Environment Variable   | Default Value     | Description |
+| ---------------------- | ----------------- | ----------- |
+| `ETCD_SERVICE_HOST`    | N/A               | The host on which the etcd service runs. |
+| `ETCD_SERVICE_PORT`    | N/A               | The etcd port number.                    |
+| `PPS_WORKER_GRPC_PORT` | `80`              | The GRPs port number.                    |
+| `PORT`                 | `650`             | The `pachd` port number. |
+| `PPROF_PORT`            | `651`             | The port number for the `pprof` tool.     |
+| `HTTP_PORT`             | `652`             | The HTTP port number.   |
+| `PEER_PORT`             | `653`             | The port for pachd-to-pachd communication. |
+| `PPS_ETCD_PREFIX`        | `pachyderm_pps`   | ???  |
```

Comment:
I don't think we want to expose this in the docs. This is the prefix we use to separate PPS data stored in etcd. We made it an environment variable to give ourselves flexibility to change it, but users aren't expected to want to modify this or care what it's set to.

Other similar variables that I don't think we want to expose:
- ETCD_PREFIX
- PFS_ETCD_PREFIX
- PACHYDERM_AUTH_ETCD_PREFIX
- PACHYDERM_ENTERPRISE_ETCD_PREFIX